.travis.yml

```diff
@@ -3,6 +3,10 @@ dist: trusty
 language: node_js
 node_js:
 - node
+services:
+- mongodb
+before_script:
+  - sleep 5
```

Comment:
1) This starts mongodb on the instance Travis ci spins up for testing
2) Have to sleep because there are some cases when mongo will not accept the connection immediately after starting. https://docs.travis-ci.com/user/database-setup/#MongoDB-does-not-immediately-accept-connections